21.py

```diff
@@ -0,0 +1,50 @@
+# https://projecteuler.net/problem=21
+
+def is_proper_divisor(divisor, n):
+    if divisor >= n:
+        return False
+    if n % divisor == 0:
+        return True
+    return False
+
+
+def sum_proper_divisors(n):
+    sum = 0
```

Comment:
sum is a function name

Reply:
yeah i'm surprised python let me do that. i'll change it anyway